cdap-app-fabric/pom.xml

```diff
@@ -122,6 +122,11 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-core_2.10</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.elasticsearch</groupId>
+      <artifactId>elasticsearch-hadoop-mr</artifactId>
+      <version>2.1.0</version>
```

Comment:
do we need this version here? I think the parent pom has the version in the dependency management?